dev/NumberBox/NumberBox.idl

```diff
@@ -66,6 +66,9 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     [MUX_DEFAULT_VALUE("false")]
     Boolean WrapEnabled;
 
+    [MUX_DEFAULT_VALUE("false")]
+    Boolean AcceptsCalculation;
```

Comment:
I keep reading this as AcceptsCalculations with an s, should this name be plural?

Reply:
The spec says AcceptCalculation, so that's what I have for now. We haven't gone through API review yet so all property names are temporary.